jdaviz/configs/mosviz/plugins/viewers.py

```diff
@@ -230,6 +242,10 @@ class MosvizProfileView(SpecvizProfileView):
                     ['jdaviz:sidebar_plot', 'jdaviz:sidebar_export']
                 ]
 
+    def set_plot_axes(self):
+        super().set_plot_axes()
+        self.figure.axes[1].num_ticks = 5
+
```

Comment:
can we safely remove this by any chance now that we override `num_ticks` elsewhere?

Reply:
But you set it to 8 elsewhere. I find 5 better for Mosviz due to how crowded it is.

Reply:
then maybe we should just set it to 5 everywhere?  I'm just trying to avoid specific overrides as much as possible that are going to be a headache to maintain in the future.... but if you think this should be 5 and the other should be 8, then this makes sense

Reply:
I do think Mosviz needs less tick marks than the others, yes.